src/main/kotlin/org/rust/ide/inspections/lints/RsLint.kt

```diff
@@ -46,7 +46,7 @@ enum class RsLint(
      */
     fun levelFor(el: PsiElement): RsLintLevel = explicitLevel(el) ?: superModsLevel(el) ?: defaultLevel
 
-    private fun explicitLevel(el: PsiElement): RsLintLevel? = el.ancestors
+    fun explicitLevel(el: PsiElement): RsLintLevel? = el.ancestors
```

Comment:
Hmm, `el.contexts` does not properly work for items expanded from macros, but can't suggest `el.contexts` here because attributes on a macro itself should also be accounted